suite-native/module-accounts-import/src/components/AccountImportImportedAccount.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import { useNavigation } from '@react-navigation/core';
+
+import { Box, Button, Divider, Text } from '@suite-native/atoms';
+import { AccountListItem } from '@suite-native/accounts';
+import { Account } from '@suite-common/wallet-types';
+import {
+    AccountsImportStackParamList,
+    AccountsImportStackRoutes,
+    AppTabsRoutes,
+    HomeStackRoutes,
+    RootStackParamList,
+    RootStackRoutes,
+    StackToTabCompositeProps,
+} from '@suite-native/navigation';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { AccountImportSummarySection } from './AccountImportSummarySection';
+
+type AccountImportImportedAccountProps = {
+    account: Account;
+};
+
+type NavigationProp = StackToTabCompositeProps<
```

Comment:
I don't think you should have this type here, this type should in `@suite-native/navigation` I think

Reply:
i will create follow up task for this and add comments to existing types (probably in separate PR to not hold this one)